.release-it.json

```diff
@@ -1,12 +1,11 @@
 {
-  "non-interactive": false,
+  "ci": false,
   "dry-run": false,
   "verbose": false,
   "force": false,
-  "scripts": {
-    "afterRelease": null,
-    "beforeStage": ["npm run check:nightly:stark-versions -- ${version}", "npm run generate:changelog"],
-    "beforeStart": ["npm run check:starter:stark-versions"]
+  "hooks": {
+    "after:bump": ["npm run check:nightly:stark-versions -- ${version}", "npm run generate:changelog"],
+    "before:init": ["npm run check:starter:stark-versions"]
```

Comment:
I suggest to put these hooks in the order in which they are executed instead of being alphabetically, otherwise is a bit confusing... what do you think?

Reply:
I think you're right! It's easier to understand 😊